conda/environments/cuxfilter_dev_cuda10.1.yml

```diff
@@ -24,6 +25,7 @@ dependencies:
 - nodejs
 - recommonmark
 - pytest
+- mock
```

Comment:
Do you need `mock` for conda builds or GPU testing? If so, you need to add it here as well - https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/rapids-build-env/meta.yaml

Reply:
@mike-wendt it's only used in a basic (non-GPU) unit testing, it is not needed in the release package or at runtime. So I think this is sufficient? But please lmk if I have misunderstood anything

Reply:
seems to have passed w/o adding anything (cc @AjayThorve )